snap/hooks/configure

```diff
@@ -4,4 +4,6 @@
 
 k8s::common::setup_env
 
-k8s::cmd::k8s x-snapd-config reconcile
+if [ -e "$SNAP_COMMON/var/lib/k8sd/state/control.socket" ]; then
```

Comment:
best to revert this, as it is a 95% solution, and 50% problem. if you `snap set` without k8sd running, your configs will be silently dropped. also, the socket existing does not mean that k8sd is ready to accept RPCs, or that cluster quorum is up for things to succeed.

in this case, the safest thing and least surprising thing to do is to just fail, so this check is unnecessary.